Bottom: 4a335549d8d6effaeaee2eed132e24589e15c369
Top:    8392b73e3e5fb691fd3bb88919f19ffbae7e0503
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 14:35:54 -0500

Refresh of add-buildbot-role

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index 68181dcc..d255a7ec 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -6,9 +6,9 @@
   become_user: root
   become_method: sudo
   tags:
-  - buildbot
-  roles:
-  - name: Build and start the Buildbot master container
-    role: buildbot-master
-    tags:
+    - buildbot
     - buildbot-master
+  tasks:
+    - name: Build and start the Buildbot master container
+      import_role:
+        name: buildbot-master
